### PR TITLE
Fix Travis-CI in the master & some cleanup of the Travis config

### PR DESCRIPTION
**Pull request by skirpichev**
* this reverts #501. I do think - these tests don't make sense for python-only library (see also #553)
* run tests on CPython 3.9

@MaxGaukler, you may want to look on this too, esp. the first patch.

### CODE REVIEW

**Comment by skirpichev** [comment]
@fredrik-johansson, this is an alternative to #552

**Comment by MaxGaukler** [comment]
As the gmpy backend outsources the integer computations to an external library, testing it on arm64 could catch architecture-dependent bugs in that library. You may want to look into whether we test both backends (gmpy and pure python).

**Comment by skirpichev** [comment]
> As the gmpy backend

I think, this belongs to the gmpy testing (and it seems - there are arm64 tests).

> You may want to look into whether we test both backends (gmpy and pure python).

Unfortunately, no.  The current config tests only pure-python arithmetics.

File: .travis.yml [hunk 1]
```diff
@@ -1,58 +1,28 @@
...
+python:
+  - 2.7
+  - 3.4
+  - 3.5
+  - 3.7
+  - 3.8
+  - 3.9-dev
+  - pypy
+  - pypy3
 matrix:
```
Comment (cclauss) [inline .travis.yml:11]:
```suggestion
jobs:
```
Matrix is deprecated in favor of Jobs.  See Travis CI build config validation

**Pull request merged**